nipype/algorithms/tests/test_stats.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+
+import numpy as np
+import nibabel as nb
+from nipype.algorithms.stats import ActivationCount
+
+
+def test_ActivationCount(tmpdir):
+    tmpdir.chdir()
+    in_files = ['{:d}.nii'.format(i) for i in range(3)]
+    for fname in in_files:
+        nb.Nifti1Image(np.random.normal(size=(5, 5, 5)),
+                       np.eye(4)).to_filename(fname)
+
+    acm = ActivationCount(in_files=in_files)
+    res = acm.run()
+    diff = nb.load(res.outputs.out_file)
+    pos = nb.load(res.outputs.acm_pos)
+    neg = nb.load(res.outputs.acm_neg)
+    assert np.allclose(diff.get_data(), pos.get_data() - neg.get_data())
```

Comment:
should we also check some statistics `pos.get_data` itself?

Reply:
The main things I can think of that should be true of performing this on some random noise would be `np.all(pos.get_data() > 0)` and `np.all(neg.get_data() > 0)`.

Reply:
here is an example: https://github.com/oesteban/nipype/pull/9/commits/6e53f016f819bd6330fa63f7d9002cd6c565432b

Reply:
That looks valid, but I'm not fully clear on what assurances this gets us for the interface. I guess, roughly, that Gaussian noise is being filtered as we'd expect?

I'm okay if this test goes in, but my overall feeling is that a useful test asserts properties that should be true given any real or simulated inputs, and this seems tied to the specific distribution of simulated random variates.

Reply:
you usually use specific cases for tests, and often some simple cases, so you know the answer.

The interface should work for any real or simulated inputs, but I'm checking here for a specific distribution (exactly the same as @oesteban used in the original test). Obviously I'm not able to predict answers for completely random set of numbers, but for the normal distribution I can.

I just thought that this adds extra checks to the original @oesteban test, that was only checking `pos-neg == diff`, but I won't insist to include it.

Reply:
we can also just generate a small array and just calculate the expected output of the interface.